tests/Command/PullRequest/PullRequestCreateCommandTest.php

```diff
@@ -132,7 +97,7 @@ public function opens_pull_request_to_a_specific_organization_or_username($args)
         $tester->execute($args, ['interactive' => false]);
 
         $res = trim($tester->getDisplay(true));
-        $this->assertContains('Making PR from '.$args['--source-org'].':issue-145 to gushphp:master', $res);
+        $this->assertContains('Opened pull request', $res);
```

Comment:
The link is still there :) I amended the commit to include the link in the expected output.
This was only to replace the expectation as you need at least one expectation.